From 121a8c494ade88e401849107596a9b6852cf8533 Mon Sep 17 00:00:00 2001
From: Brian Zhu <brian.zhu@amlogic.com>
Date: Thu, 28 Jan 2021 00:26:24 +0800
Subject: [PATCH 2/2] di: temp fix for di interface [3/3]

PD#SWPL-37346

Problem:
buffer lock

Solution:
remove it

Verify:
Verified on ab311

Change-Id: I1d04da0932b7162536f6b32e8435bdf4f4c7f4b3
Signed-off-by: Brian Zhu <brian.zhu@amlogic.com>
---
 drivers/amlogic/media/di_multi/di_task.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/amlogic/media/di_multi/di_task.c b/drivers/amlogic/media/di_multi/di_task.c
index b2d5afd4b4c4..a84b17fed5a5 100644
--- a/drivers/amlogic/media/di_multi/di_task.c
+++ b/drivers/amlogic/media/di_multi/di_task.c
@@ -142,7 +142,7 @@ bool task_send_cmd2(unsigned int ch, unsigned int cmd)
 
 		PR_ERR("%s:lost cmd[%d]\n", __func__, val);
 		tsk->err_cmd_cnt++;
-		/*return false;*/
+		return false;
 	}
 	kfifo_in_spinlocked(&tsk->fifo_cmd2[ch], &cmd, sizeof(unsigned int),
 			    &tsk->lock_cmd2[ch]);
-- 
2.31.1

